src/models/git-hub-server-app.ts

```diff
@@ -5,21 +5,25 @@ const encrypted = EncryptedField(Sequelize, process.env.STORAGE_SECRET);
 
 interface GitHubServerAppPayload {
 	uuid: string;
```

Comment:
could you please update the file name? it's bothering me, everywhere else `github` is one word